deblur/workflow.py

```diff
@@ -417,15 +417,15 @@ def remove_artifacts_seqs(seqs_fp,
 
     if coverage_thresh is None:
         if negate:
-            coverage_thresh = 0.95
+            coverage_thresh = 0.95 * 100
         else:
-            coverage_thresh = 0.3
+            coverage_thresh = 0.6 * 100
```

Comment:
the problem is that sometimes it can go for a shorter alignment with higher identity compared to longer alignment with lower identity (i.e. maybe 50% coverage 100% identity instead of 100% coverage 80% identity)... I'm not sure how it chooses...